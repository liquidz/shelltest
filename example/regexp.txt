bash$ echo foobar
=~ foo
bash$ echo foobar
=~ bar
bash$ echo foobar
=~ foobar
