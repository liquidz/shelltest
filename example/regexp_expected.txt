$ shelltest example/regexp.txt
...

3 tests, 0 failures
